src/Pages/MembershipApplication/MembershipPlan.js

```diff
@@ -162,6 +163,11 @@ class MembershipPlan extends Component {
             </div>
           </div>
         </Container>
+        <Button onClick={() => {
+          sendVerificationEmail('evanuxd@gmail.com', 'evan');
```

Comment:
nice email xd

Reply:
whoops forgot to remove this, thanks again